core/src/service/crypto_service.rs

```diff
@@ -14,16 +15,17 @@ use self::rand::RngCore;
 use self::rsa::hash::Hashes;
 use self::rsa::{PaddingScheme, PublicKey, RSAPrivateKey, RSAPublicKey};
 
-#[derive(PartialEq, Debug, Clone)]
+#[derive(PartialEq, Debug, Clone, Deserialize, Serialize)]
 pub struct EncryptedValue {
     pub garbage: String,
 }
 
-#[derive(PartialEq, Debug)]
+#[derive(PartialEq, Debug, Deserialize, Serialize)]
```

Comment:
`DecryptedValue` shouldn't get `Serialize`d anywhere

Reply:
So how does this go over FFI then?

Reply:
I don't see any benefit to no deserializing/serializing it when `.secret` is essentially the same thing. Being able to say we don't `Serialize` `DecryptedValue` doesn't actually _mean_ anything